src/Symfony/Component/Cache/Adapter/PhpFilesAdapter.php

```diff
@@ -31,8 +31,8 @@ public function __construct(string $namespace = '', int $defaultLifetime = 0, st
         self::$startTime = self::$startTime ?? $_SERVER['REQUEST_TIME'] ?? time();
         parent::__construct('', $defaultLifetime);
         $this->init($namespace, $directory);
-
-        $e = new \Exception();
-        $this->includeHandler = function () use ($e) { throw $e; };
+        $this->includeHandler = function ($type, $msg, $file, $line) {
+            throw new \ErrorException($msg, 0, $type, $file, $line);
```

Comment:
Throwing an ErrorException instead of an Exception will change what happens inside `doFetch` as `ErrorException` gets catched there. Is it breaking things or fixing them ?

Reply:
well, actually, the catch I looked at is changed in this PR...